client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -1,152 +1,217 @@
 import { Button, Gridicon } from '@automattic/components';
+import { Icon, info } from '@wordpress/icons';
 import classnames from 'classnames';
-import { localize } from 'i18n-calypso';
-import { flowRight as compose, includes } from 'lodash';
-import { Component } from 'react';
-import { connect } from 'react-redux';
+import { useTranslate, getLocaleSlug } from 'i18n-calypso';
+import { includes } from 'lodash';
+import { useMemo, useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import semverGt from 'semver/functions/gt';
```

Comment:
Can `version_compare` 
https://github.com/Automattic/wp-calypso/blob/ea33197d3e2c9acd0b1cf674af577ed98114c98f/client/lib/version-compare/index.js#L7

accomplish the same thing without introducing a new dependency?

Reply:
That is right, thanks for pointing it.

fixed https://github.com/Automattic/wp-calypso/pull/58866/commits/d2c00f0f0a3036b0c3f0dc9d65b8c5298b3df09d